src/ptvsd/wrapper.py

```diff
@@ -1423,23 +1423,7 @@ def on_threads(self, request, args):
 
     @async_handler
     def on_source(self, request, args):
-        """Request to get the source"""
-        source_reference = args.get('sourceReference', 0)
-
-        if source_reference == 0:
-            self.send_error_response(request, 'Source unavailable')
-        else:
-            pydevd_request = copy.deepcopy(request)
-            del pydevd_request['seq']  # A new seq should be created for pydevd.
-            _, _, resp_args = yield self.pydevd_request(
-                pydevd_comm.CMD_LOAD_SOURCE,
-                pydevd_request,
-                is_json=True)
-
-            if resp_args.get('success', True):
-                self.send_response(request, **resp_args['body'])
-            else:
-                self.send_error_response(request, resp_args.get('message', 'Error retrieving source'))
+        self._forward_request_to_pydevd(request, args)
```

Comment:
This should no longer be `async_handler`.

Reply:
oops